pkg/driver/node.go

```diff
@@ -660,3 +682,45 @@ func hasMountOption(options []string, opt string) bool {
 	}
 	return false
 }
+
+// isMounted checks if target is mounted. It does NOT return an error if target
+// doesn't exist.
+func (d *nodeService) isMounted(target string) (bool, error) {
+	/*
+		Checking if it's a mount point using IsLikelyNotMountPoint. There are three different return values,
+		1. true, err when the directory does not exist or corrupted.
+		2. false, nil when the path is already mounted with a device.
+		3. true, nil when the path is not mounted with any device.
```

Comment:
Maybe this description can be placed above function declaration.
Also: 
> 1. true, err when the directory does not exist or corrupted

Seems to provide an error but this seems contrary to
> It does NOT return an error if target doesn't exist.

Reply:
The comments explain the output of `IsLikelyNotMountPoint`, not the output of `isMounted`.